includes/class-wc-payments-payment-request-button-handler.php

```diff
@@ -761,6 +761,13 @@ public function scripts() {
 
 		WC_Payments::register_script_with_dependencies( 'WCPAY_PAYMENT_REQUEST', 'dist/payment-request', [ 'jquery', 'stripe' ] );
 
+		wp_enqueue_style(
+			'WCPAY_PAYMENT_REQUEST',
+			plugins_url( 'dist/payment-request.css', WCPAY_PLUGIN_FILE ),
+			[],
+			'1.0'
+		);
```

Comment:
To maintain consistency with the rest of the plugin, I believe we should use:

```suggestion
		WC_Payments_Utils::register_style(
			'WCPAY_PAYMENT_REQUEST',
			plugins_url( 'dist/payment-request.css', WCPAY_PLUGIN_FILE ),
			[],
			WC_Payments::get_file_version( 'dist/payment-request.css' )
		);
```